gengokit/httptransport/templates_test.go

```diff
@@ -62,6 +62,8 @@ func TestGenClientEncode(t *testing.T) {
 // the http request (path, query, and body).
 func EncodeHTTPSumZeroRequest(_ context.Context, r *http.Request, request interface{}) error {
 	fmt.Printf("Encoding request %v\n", request)
+	strval := ""
+	_ = strval
```

Comment:
These variables are declared this way because they're variables that need to exist, but depending on the types of the variables we decode in this method, code may not be generated which ever actually uses these variables, which would cause the compiler to fail to compile this generated code. So instead, we must declare them and then tell the compiler to not complain if that variable is ever unused by assigning it to underscore.